pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/PersistentTopics.java

```diff
@@ -662,6 +662,32 @@ public PersistentOfflineTopicStats getBacklog(@PathParam("property") String prop
         return internalGetBacklog(authoritative);
     }
 
+    @GET
+    @Path("/{property}/{cluster}/{namespace}/{topic}/messageId/{messageId}/checkLedgerExists/{checkLedgerExists}/backlogSize")
```

Comment:
Do we need to support V1 for new apis like this?

Reply:
delete V1